### ISSUE TITLE
Shinyforms compared to concerto platform

### ISSUE THREAD

**Issue by rdatasculptor** [daattali/shinyforms#23]
First of all: I think this package is a great idea! 

I was wondering, ar you aware of the existence of Concerto platform? R-based and also a wat to publish forms on the net. Do you think that shinyforms could, in potention, evolve into something similar? To my opinion Concerto acted rather slow (and I must admit, that was two versions ago), so any approvement on things like speed would be very welcome. Also I think that in the end a well performing, secure connected, safe, open source testrunner application with R as its score calculator could cause a revolution in the world of assessments. It would be very easy to start as a new and competing test publisher. 

Just some thoughts.

Kind regards

**Comment by rdatasculptor** [comment]
(I said approvements, but ofcourse I meant improvements)

**Comment by daattali** [comment]
I did not know about concerto. It looks like it's a web service (not using R-Shiny, just a regular webtool) that lets you do surveys, and it uses R for statistical analysis?  Correct me if I'm wrong, I only took a very quick look.

I envisioned shinyforms to more of of a way to ask for form inputs (like Google forms), rather than a survey of one question at a time, though that can certainly be added later on as an improvement.  Unfortunately this package is still on hold while I'm working on some other things (I have a few other package ideas that are much simpler and I'm working on), but I do plan on coming back to make this package awesome later.

Thanks for your thoughts!
